components/ai_chat/core/browser/constants.h

```diff
@@ -31,6 +31,11 @@ extern const char kLeoModelSupportUrl[];
 inline constexpr int kCustomModelMaxPageContentLength = 9600;
 inline constexpr int kCustomModelLongConversationCharLimit = 10000;
 
+inline constexpr char kBraveSearchURLPrefix[] = "search";
+inline constexpr char kBraveSearchSERPPath[] = "/search";
+inline constexpr char kBraveSearchQuerySummaryAPIPath[] =
```

Comment:
nit: do these two have to be in a common constants file considering they're only used in one place? Up to you.

Reply:
Changed to literal strings in the final squashed commit.